README.md

```diff
@@ -17,6 +17,21 @@ oc scale --replicas=0 deploy/cluster-storage-operator -n openshift-cluster-stora
 oc -n openshift-cluster-csi-drivers delete deployment.apps/powervs-block-csi-driver-operator deployment.apps/ibm-powervs-block-csi-driver-controller daemonset.apps/ibm-powervs-block-csi-driver-node
 ```
 
+# Define custom endpoints for the CSI driver:
+Include a [serviceEndpoints](https://docs.openshift.com/container-platform/4.17/rest_api/config_apis/infrastructure-config-openshift-io-v1.html#status-platformstatus-powervs-serviceendpoints-2) section to configure non-default endpoints. For example:
```

Comment:
User wont be able to directly update the status subresoruce, They need to add it in spec we have a controlller which does some basic validation and mirros it across the status.

So you should change this link to point to spec: https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L1717-L1724

Reply:
Hey, thanks for pointing out. 
I have updated the URL to : https://docs.openshift.com/container-platform/4.17/rest_api/config_apis/infrastructure-config-openshift-io-v1.html#spec-platformspec-powervs-serviceendpoints-2 which does cover the `spec`. 
I'll also all the reference to : https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L1717-L1724